Rakefile

```diff
@@ -309,10 +313,9 @@ REGEN_METADATA = {
         tag: 'arm_recovery'
     },
     azure_mgmt_recovery_services_backup: {
-        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-recoveryservicesbackup/compositeRecoveryServicesBackupClient.json',
+        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-recoveryservicesbackup/readme.md',
```

Comment:
It looks like you are solving this problem only for local regen story. What about the AutoPRs?

Apart from this change, we should now use the [swagger_to_sdk](https://github.com/lmazuel/swagger-to-sdk) version 2.0. I believe that it supports autorest with .json / .md.

We should refer https://github.com/Azure/azure-sdk-for-python/blob/master/swagger_to_sdk_config.json#L3 to fix that as well along with that to be consistent.